### PR TITLE
Replace nginx v1.9.5 with v1.9.6 and metrics-server v0.6.4 with v0.7.0

### PR DESCRIPTION
**Pull request by chiukapoor**
### Update
* Replace nginx v1.9.5 with v1.9.6 and metrics-server v0.6.4 with v0.7.0

### Related
* https://github.com/rancher/rancher/issues/43109

### CODE REVIEW

File: pkg/rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -9965,10 +9965,10 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			AciOpenvSwitchContainer:   "noiro/openvswitch:6.0.3.2.81c2369",
 			AciControllerContainer:    "noiro/aci-containers-controller:6.0.3.2.81c2369",
 			PodInfraContainer:         "rancher/mirrored-pause:3.7",
-			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.9.5-rancher1",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.9.6-rancher1",
 			IngressBackend:            "rancher/mirrored-nginx-ingress-controller-defaultbackend:1.5-rancher1",
 			IngressWebhook:            "rancher/mirrored-ingress-nginx-kube-webhook-certgen:v20231011-8b53cabe0",
```
Comment (kinarashah) [inline pkg/rke/k8s_rke_system_images.go]: We need to update webhook certgen as well, 1.9.6 uses latest release `v20231226-1a7112e06` https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.9.6

File: pkg/rke/templates/metrics_v0.7.0.go [hunk 2]
```diff
@@ -187,7 +198,7 @@ spec:
           periodSeconds: 10
         args:
         - --cert-dir=/tmp
-        - --secure-port=4443
+        - --secure-port=10250
```
Comment (kinarashah) [inline pkg/rke/templates/metrics_v0.7.0.go:201]: Kubelet uses `10250` but looks like the change in upstream was intentional `https://github.com/kubernetes/kubernetes/pull/105957#issuecomment-953809831`

File: pkg/rke/k8s_rke_system_images.go (hunk 1 above)
Comment (chiukapoor) [reply pkg/rke/k8s_rke_system_images.go]: Created image-mirror [PR](https://github.com/rancher/image-mirror/pull/581) to add the mirrored image.

File: pkg/rke/templates/metrics_v0.7.0.go (hunk 2 above)
Comment (chiukapoor) [reply pkg/rke/templates/metrics_v0.7.0.go:201]: k8s `v1.28.6` deployed successfully on Rancher with metrics-server `v0.7.0`
Screenshot from 2024-02-22 01-31-46

**Review by kinarashah — APPROVED** [review]

**Pull request merged**